### PR TITLE
Update createRoom to support creating DM rooms

### PR DESCRIPTION
**Pull request by dbkr**
* Make ChatInviteDialog and MemberInfo createRoom use it
* Fix bug in setDMRoom

https://github.com/vector-im/vector-web/issues/2099

### CODE REVIEW

File: src/createRoom.js [hunk 1]
```diff
@@ -46,10 +48,15 @@ function createRoom(opts) {
         return q(null);
     }
 
+    const defaultPreset = opts.dmUserId ? 'trusted_private_chat' : 'private_chat';
```
Comment (ara4n) [inline src/createRoom.js]: hm, are these really the right presets?

**Comment by ara4n** [comment]
other than q, lgtm

File: src/createRoom.js (hunk 1 above)
Comment (dbkr) [inline src/createRoom.js]: I think trusted_private_chat was intended for DMs

**Comment by ara4n** [comment]
lgtm

**Pull request merged**